solr/solrj/src/java/org/apache/solr/client/solrj/impl/LBHttpSolrClient.java

```diff
@@ -382,13 +385,14 @@ public ResponseAndException request(Req req) throws SolrServerException, IOExcep
         .filter(r -> null == r.getEx())
         .findAny();
 
+
     if(aliveResponseAndException.isPresent()){
       return aliveResponseAndException.get();
     }
     else{
       Optional<ResponseAndException> deadResponseAndException = skipped.parallelStream()
         .map(wrapper -> {
-          return getResponseAndException(req, isNonRetryable, true, wrapper.getKey(), wrapper.client);
+          return  getResponseAndException(req, isNonRetryable, true, wrapper.getKey(), wrapper.client);
         })
           .filter(r -> null == r.getEx())
```

Comment:
Missing null check for getResponseAndException?

Reply:
added